### PR TITLE
Add Legally Sensitive Flag

### PR DESCRIPTION
**Pull request by susiecoleman**
Media Atoms can now be marked as legally sensitive. 

videofilter

This flag has been added to the content-atom thrift definition https://github.com/guardian/content-atom/pull/54/

### CODE REVIEW

File: project/BuildVars.scala [hunk 1]
```diff
@@ -2,11 +2,11 @@ import sbt._
 
 object BuildVars {
   lazy val awsVersion         = "1.11.48"
-  lazy val contentAtomVersion = "2.4.14"
+  lazy val contentAtomVersion = "2.4.16"
```
Comment (akash1810) [inline project/BuildVars.scala]: eeeeep! Would me merging in https://github.com/guardian/media-atom-maker/pull/108/files#diff-a7d6ef61e0ad0403cd2d8180afdcbe42R5 cause pain?

Comment (akash1810) [inline project/BuildVars.scala]: actually don't think it should as the fields you're adding are Optional - I'll merge #108 now

File: public/video-ui/src/components/VideoEdit/ContentFlags.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React, {PropTypes} from 'react';
```
Comment (akash1810) [inline public/video-ui/src/components/VideoEdit/ContentFlags.js]: don't seem to be using `PropTypes`?

File: public/video-ui/src/components/VideoEdit/ContentFlags.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    const flag = {[e.target.id]: e.target.checked}
+    const video = Object.assign({}, this.props.video, flag);
+    this.props.saveAndUpdateVideo(video)
+  }
+
+  render(){
+    return(
+      <div>
+      <p className="details-list__title">Filters</p>
+        <table className="details-list">
```
Comment (akash1810) [inline public/video-ui/src/components/VideoEdit/ContentFlags.js]: 😱  why `table`?

Comment (susiecoleman) [inline public/video-ui/src/components/VideoEdit/ContentFlags.js]: It seemed sensible when there was going to be lots of switches...Simplified the code

File: public/video-ui/src/components/VideoEdit/ContentFlags.js (hunk 2 above)
Comment (susiecoleman) [inline public/video-ui/src/components/VideoEdit/ContentFlags.js]: Removed

File: public/video-ui/src/components/VideoEdit/ContentFlags.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    const video = Object.assign({}, this.props.video, flag);
+    this.props.saveAndUpdateVideo(video)
+  }
+
+  render(){
+    return(
+      <div>
+        <p className="details-list__title">Filters</p>
+        <p className="details-list__title">Legally sensitive</p>
+        <div className="details-list__labled-filter">
```
Comment (akash1810) [inline public/video-ui/src/components/VideoEdit/ContentFlags.js]: lol `labled` 👦 ➡️

Comment (susiecoleman) [inline public/video-ui/src/components/VideoEdit/ContentFlags.js]: Dammit 😞  spell checker needed

**Pull request merged**